### PR TITLE
Date json encode format

### PR DESCRIPTION
**Pull request by ADmad**
Refs #93

### CODE REVIEW

**Comment by ADmad** [comment]
That's strange, I don't get any failures locally.

**Comment by ADmad** [comment]
No failures on appveyor either so it seem problem is with travis' ICU lib version. Btw i have ubuntu 14.04.

**Comment by ADmad** [comment]
It is an issue with outdated ICU on travis VMs https://github.com/travis-ci/travis-ci/issues/3616

**Comment by markstory** [comment]
@ADmad Do you want to try the php nightly build?

**Comment by ADmad** [comment]
@markstory Doesn't work. I am following up travis' issue tracker.

**Comment by ADmad** [comment]
Newer ICU version is unlikely to be available on stable PHP version on travis anytime soon. So I just put a check to skip relevant tests based on ICU version. They will still run on appveyor so I think we are good.

File: tests/TestCase/I18n/DateTest.php [hunk 1]
```diff
@@ -32,6 +32,28 @@ class DateTest extends TestCase
...
+     * setUpBeforeClass
+     *
+     * @return void
+     */
+    public static function setUpBeforeClass()
+    {
+        ob_start();
+        phpinfo();
+        $phpinfo = ob_get_clean();
+        preg_match('/ICU version => ([\d\.]+)/', $phpinfo, $matches);
```
Comment (markstory) [inline tests/TestCase/I18n/DateTest.php]: Should we do this as part of the test suite `bootstrap.php`? Might be a nice to have a constant with the ICU version.

**Comment by ADmad** [comment]
@markstory Updated to use already available constant :)

**Pull request merged**